css-three-column-layout/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <!-- row 1 -->
+    <h1>Designed for life science. Optimized for SaMD and Digital Health</h1>
+    <!-- row 2 -->
+    <div class="row">
+      <!-- column 1 -->
+      <div class="column column-one">
+        <!-- row 1 - logo -->
+        <div class="logo-container">
+          <img src="images/merge.png" alt="merge logo">
+        </div>
+        <!-- row 2 - header -->
+        <h2>Streamlined QMS</h2>
+        <!-- row 3 - text -->
+        <p>A QMS that includes modules for all stages of the product life cycle from Design Control to CAPA</p>
+      </div>
+      <!-- column 2 -->
+      <div class="column column-two">
+        <!-- row 1 - logo -->
+        <div class="logo-container">
+          <img src="images/cloud-computing.png" alt="cloud computing logo">
+        </div>
+        <!-- row 2 - header -->
+        <h2>Integrates with existing tools</h2>
+        <!-- row 3 - text -->
+        <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+      </div>
+      <!-- column 3 -->
+      <div class="column column-three">
+        <!-- row 1 - logo -->
+        <div class="logo-container">
+          <img src="images/discussion.png" alt="discussion logo">
+        </div>
+        <!-- row 2 - header -->
+        <h2>In-depth Onboarding</h2>
+        <!-- row 3 - text -->
+        <p>We train your entire team and provide you with support when and as it's needed</p>
+      </div>
```

Comment:
Please remove the code comments here. Though I appreciate the information they provide, is makes reading the code more difficult.

Reply:
My bad! I should've deleted them since it was more helpful for me to organize and visualize the grid